src/Generators/ModelGenerator.php

```diff
@@ -259,7 +243,7 @@ protected function isRequired(string $typeName): bool
 
     protected function getRelationType(string $model, string $relation): string
     {
-        if (in_array($relation, self::PLURAL_NUMBER_REQUIRED)) {
+        if (in_array($relation, ['belongsToMany', 'hasMany'])) {
```

Comment:
let's implement the separate helper for it 


```suggestion
        if ($this->isPluralRelation($relation)) {
```